python/requirements_compiled.txt

```diff
@@ -1780,13 +1789,13 @@ python-lsp-jsonrpc==1.0.0
     # via semgrep
 python-multipart==0.0.6
     # via gradio
-python-snappy==0.6.1 ; python_version <= "3.10"
-    # via -r /ray/ci/../python/requirements/ml/data-test-requirements.txt
-pytorch-lightning==1.6.5
-    # via -r /ray/ci/../python/requirements/ml/tune-test-requirements.txt
+pytorch-lightning==1.9.5
```

Comment:
@can-anyscale looks like this got bumped back up to 1.9

Reply:
w000t, good catch